docs/tags-are-not-allowed-for-proxy-resources.md

```diff
@@ -0,0 +1,74 @@
+# TagsAreNotAllowedForProxyResources
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Put-V1-30
+
+## Description
+
+Validates that `tags` is not defined in the properties bag, but rather as a top-level property,
+`tags` should not be specified in the properties bag for proxy resources. Consider using a Tracked resource instead.
+
+## How to fix the violation
+
+Ensure that any `tags` definitions are as top-level properties, not in the properties bag.
+
```

Comment:
Either remove the 'tags' property from the properties bag or consider using a tracked resource.